src/main/java/uk/ac/cam/cl/dtg/isaac/dao/EventBookingPersistenceManager.java

```diff
@@ -188,7 +188,7 @@ public boolean isUserBooked(final String eventId, final Long userId) throws Segu
         try {
             final EventBooking bookingByEventAndUser = dao.findBookingByEventAndUser(eventId, userId);
             List<BookingStatus> bookedStatuses = Arrays.asList(
-                    BookingStatus.CONFIRMED, BookingStatus.ATTENDED, BookingStatus.ABSENT);
+                    BookingStatus.CONFIRMED, BookingStatus.ATTENDED, BookingStatus.ABSENT, BookingStatus.RESERVED);
```

Comment:
We should be careful about the meaning of this method. Before it meant that a user had a confirmed booking. Are we defining reserved as definitely confirmed now?

Reply:
Good point – I saw it as a way of retrieving users that are associated with an event and blissfully ignored any implied meaning. We could have a separate method for reservations and do both checks only where necessary.